### ISSUE TITLE
Don't allow edited transcripts to be replaced

### ISSUE THREAD

**Issue by Saltzshaker** [popuparchive/pop-up-archive#1316]

**Comment by AndersEverett** [comment]
See best_transcript method. Add hierarchy: basic => premium => (updated at > created at).

**Comment by AndersEverett** [comment]
5b8010164d Adds edited transcript to the top of the selection hierarchy.

**Comment by Saltzshaker** [comment]
@AndersEverett how can I test this? Since processing isn't working in staging, I can't refire a transcript to see if it does/doesn't replace an edited one.

**Comment by AndersEverett** [comment]
@Saltzshaker I'm not sure. We could try finding an audiofile in staging that has multiple transcripts; then edit one of the older transcripts to see if it will then be the one rendered.

**Comment by Saltzshaker** [comment]
Code works by looking to see if transcript has an updated_at value which is greater than the created_at value. However, changing updated_at value for an older transcript does not change which transcript displays in the UI. (Tested with https://pop-up-archive-staging.herokuapp.com/collections/522/items/5244 and https://pop-up-archive-staging.herokuapp.com/superadmin/audio_files/6600)

Also, I tried editing a transcript and the transcript updated_at value didn't change, so we might need a different approach to determining whether a transcript has been edited.

**Comment by Saltzshaker** [comment]
Since using updated_at doesn't currently work to determine whether a transcript has been edited, there are some other options:

1. Create a rake task to update Transcript updated_at value based on most recently updated TimedText. (More time-intensive to code) 
2. Update updated_at of parent object when a TimedText is updated (Easier to code, but won't work retroactively)

**Comment by annewootton** [comment]
Cool. I guess the concern with #2 not being retroactive would be that if
Studs Terkel, say, has an edited transcript from last year, and we
accidentally fire a new premium transcript for them, it would replace the
edited transcript. Yeah?

**Comment by Saltzshaker** [comment]
@annewootton Yep. But given how infrequently this happens anyway, it might be good enough to settle for fixing it with future transcripts if it means much less dev time. We can reassess while going over Megan after the holidays.

**Comment by AndersEverett** [comment]
Fix timed_text / Transcript relationship, so when timed_text updated_at  is changed, then transcript updated_at is also changed.

Add rake task that changes updated_at for all transcripts with updated_at diffs in children timed_texts.
